examples/00_quick_start/fbt_movielens.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
I think the description is not accurate for this methodology and needs to be clarified. What is described as "frequently bought together" would require generating recommendations for (user, item) pairs. What you implemented is more like "items frequently bought together with one of the items this user has bought in the past".

Reply:
I will address the description difference in the notebook. Thanks for the feedback.